python/Ganga/GPIDev/Lib/Registry/PrepRegistry.py

```diff
@@ -14,14 +14,9 @@
 
 class PrepRegistry(Registry):
 
-    def __init__(self, name, doc, dirty_flush_counter=10, update_index_time=30, dirty_max_timeout=60, dirty_min_timeout=30):
-
-        super(PrepRegistry, self).__init__(name, doc, dirty_flush_counter, update_index_time, dirty_max_timeout, dirty_min_timeout)
-
-        self.releaseThread = threading.Thread(target=self.trackandRelease, args=())
-        self.releaseThread.daemon = True
-        self.releaseThread.start()
```

Comment:
As far as I can tell, this code hasn't actually been run in several releases now due to `_dirtyModTime` and `_flushLastTime` not being set to anything any more. I don't think we've had any complaints about it have we?